lib/pg_query/deparse.rb

```diff
@@ -99,6 +99,7 @@ def deparse_item(item, context = nil) # rubocop:disable Metrics/CyclomaticComple
 
     def deparse_rangevar(node)
       output = []
+      output << 'ONLY' if node['inhOpt'] == 0
```

Comment:
Because I didn't know about this keyword, here's an explanation from the pg docs:

> If ONLY is not specified, matching rows are also updated in any tables inheriting from the named table